net_utils.js

```diff
@@ -421,3 +428,32 @@ exports.get_ipany_re = function (prefix, suffix, modifier) {
         modifier
     );
 };
+
+exports.ipv6_reverse = function(ipv6){
+    ipv6 = new IPv6.Address(ipv6);
```

Comment:
This will not work, because `ip-address` make it to full lenght other than `ipaddr.js`

```
ipaddr.js - toNormalizedString: 2001:db8:8:66:0:0:0:1
ip-address - canonicalForm: 2001:0db8:0008:0066:0000:0000:0000:0001 << needed to reverse
```